tests/pytorch/test_fused_attn.py

```diff
@@ -192,14 +198,24 @@ def _is_unfused_attention_supported(config: ModelConfig) -> bool:
     param_types.append(torch.bfloat16)
 param_types_lean = [torch.bfloat16]
 
+def get_swa(seq_q, seq_kv, w=None):
+    if w is None:
+        w = torch.randint(0, seq_kv, [2], dtype=torch.int32, device="cuda")
+    m = torch.ones(seq_q, seq_kv, dtype=torch.bool, device="cuda")
+    mu = torch.triu(m, diagonal=seq_kv-seq_q-w[0])
+    ml = torch.tril(mu, diagonal=seq_kv-seq_q+w[1])
+    ml = ~ ml
+    return w, ml
+
 @pytest.mark.skipif(_cudnn_version() < (8,9,1), reason="cuDNN 8.9.1+ is required.")
 @pytest.mark.parametrize("dtype", param_types)
 @pytest.mark.parametrize("model_configs", [model_configs_base])
 @pytest.mark.parametrize("model", model_configs_base.keys())
 @pytest.mark.parametrize("ckpt_attn", [False])
 @pytest.mark.parametrize("workspace_opt", [True, False])
 @pytest.mark.parametrize("qkv_layout", [None])
-def test_dot_product_attention(dtype, model_configs, model, ckpt_attn, workspace_opt, qkv_layout):
+@pytest.mark.parametrize("swa", [False])
```

Comment:
Why are we testing just the `False` case?

Reply:
We are testing sliding window in the `test_dpa_sliding_window()` tests so I thought there was no need to duplicate the tests here.